test/Core.ts

```diff
@@ -343,6 +343,88 @@ export class Foo {
         })
       })
 
+      it('should replace imports from the project with the local source folder (single quotes)', async () => {
+        const state = prefixWithCwd({
+          'package.json': defaultPackageJson,
+          'src/utils/foo.ts': `
+/**
+ * @since 0.0.1
+ */
+
+/**
+ * @category utils
+ * @since 0.0.1
+ */
+export class Foo {
+  /**
+   * @example
+   * import * as Foo from '${JSON.parse(defaultPackageJson)['name']}'
+   *
+   * @since 0.0.1
+   */
+  public bar(): string {
+    return 'bar'
+  }
+}
+          `
+        })
+        const capabilities = makeCapabilities(state)
+
+        assertRight(await Core.main(capabilities)(), (result) => {
+          const exampleFileName = `${[...process.cwd().split('/'), 'src', 'utils', 'foo.ts'].join(
+            '-'
+          )}-Foo-method-bar-0.ts`
+          const exampleFilePath = `${process.cwd()}/docs/examples/${exampleFileName}`
+          const importFromRegex = new RegExp("from '../../src'")
+
+          assert.strictEqual(result, undefined)
+          assert.strictEqual(command, 'ts-node')
+          assert.strictEqual(executablePath.includes('docs/examples/index.ts'), true)
+          assert.strictEqual(importFromRegex.test(fileSystemState[exampleFilePath]), true)
+        })
+      })
+
+      it('should replace imports from the project with the local source folder (double quotes)', async () => {
+        const state = prefixWithCwd({
+          'package.json': defaultPackageJson,
+          'src/utils/foo.ts': `
+/**
+ * @since 0.0.1
+ */
+
+/**
+ * @category utils
+ * @since 0.0.1
+ */
+export class Foo {
+  /**
+   * @example
+   * import * as Foo from "${JSON.parse(defaultPackageJson)['name']}"
+   *
+   * @since 0.0.1
+   */
+  public bar(): string {
+    return 'bar'
+  }
+}
+          `
+        })
+        const capabilities = makeCapabilities(state)
+
+        assertRight(await Core.main(capabilities)(), (result) => {
+          const exampleFileName = `${[...process.cwd().split('/'), 'src', 'utils', 'foo.ts'].join(
```

Comment:
I'm not so sure if it's generally a good thing to have the full absolute paths inside the `FileSystemState` object. As that's also not the case in the real environment. But I kept this as it is.

Reply:
I actually disagree with this - I cannot remember exactly what our rationale was behind including `process.cwd()` in our implementation (😅 ), but I seem to remember it being required for certain aspects of our tests to work properly. In addition, I prefer to be explicit anyways when reading and writing files to a file system.